### PR TITLE
Translate to portuguese: path related terms

### PR DESCRIPTION
**Pull request by villares**
## Author: 

@villares - Alexandre Villares

## Language: 

- Portuguese (from Brazil), used pt for Portuguese. 

## Terms defined:

-  absolute path 
-  relative path
-  current working directory
   (note: the abbreviated "working directory" form seems more common in pt, so that's what I used,
    I can change this if requested.)

## Editor

 @fmichonneau and @beatrizmilz 
  
  I would like to suggest an improvement to the CWD definition in English, mentioning the acronym and that it is often used in the context of CLI (command line interfaces). May I open another pull request for that?

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -29,6 +29,12 @@
       Una ruta que dirige a la misma ubicación en el sistema de archivos
       independientemente del contexto donde sea evaluada. Una ruta absoluta es
       el equivalente a la latitud y longitud en geografía.
+  pt:
+    term: "caminho absoluto"
+    def: >
+      Um caminho que leva ao mesmo local no sistema de arquivos
+      independente do contexto em que é avaliado. Um caminho absoluto
+      pode ser comparado com a latitude e longitude em geografía.
```
Comment (beatrizmilz) [inline glossary.yml:37]: Geograf**i**a

File: glossary.yml [hunk 2]
```diff
@@ -1681,7 +1692,17 @@
       location, such as the [current working
       directory](#current_working_directory). A relative path is the
       equivalent of giving directions using terms like "straight" and
-      "left".  ref: - absolute_path
+      "left".
+    ref: - absolute_path
```
Comment (beatrizmilz) [inline glossary.yml:1696]: Looks like the build is generating an error in this line

File: glossary.yml (hunk 1 above)
Comment (villares) [reply glossary.yml]: Obrigado!

File: glossary.yml (hunk 2 above)
Comment (villares) [reply glossary.yml:1696]: Thank you so much. I thought it was wrong, I'll  move it back!

Comment (beatrizmilz) [reply glossary.yml:1696]: You are welcome! I'm still trying to understand the patterns of yaml. 

In the lines 1704 and 1705 it looks the same pattern and its working. 

```
  ref: 
    - absolute_path
```

Comment (villares) [reply glossary.yml]: actually, first I'll try to emulate other entries that have ref at the end on the same indentation level as languages.

**Review by beatrizmilz — COMMENTED** [review]
@zkamvar  I reviewed this PR, in terms of language is  well translated to Portuguese :)

**Comment by zkamvar** [comment]
> @zkamvar I reviewed this PR, in terms of language is well translated to Portuguese :)

Gotcha! I think it just needed an approving review, but I'll get to that once I fix the merge conflicts.

**Review by zkamvar — APPROVED** [review]

File: glossary.yml [hunk 3]
```diff
@@ -66,6 +66,14 @@
...
       independientemente del contexto donde sea evaluada. Una ruta absoluta es
       el equivalente a la latitud y longitud en geografía.
+  pt:
+    term: "caminho absoluto"
+    def: >
+      Um caminho que leva ao mesmo local no sistema de arquivos
+      independente do contexto em que é avaliado. Um caminho absoluto
+      pode ser comparado com a latitude e longitude em geografia.
+  ref:
+    - relative_path
```
Comment (zkamvar) [inline glossary.yml:76]:
```suggestion
```

File: glossary.yml [hunk 4]
```diff
@@ -3575,6 +3588,15 @@
...
       "left".
+  pt:
+    term: "caminho relativo"
+    def: >
+      Um caminho cujo destino é interpretado de maneira relativa a outro
+      local, como o [diretório de trabalho](#current_working_directory). Um caminho
+      relativo é o equivalente a indicar um destino com termos como "siga em frente" 
+      e "vire a esquerda".
+  ref: 
+    - absolute_path
```
Comment (zkamvar) [inline glossary.yml:3599]:
```suggestion
```

**Comment by zkamvar** [comment]
Thank you both!

**Comment by elletjies** [comment]
@all-contributors please add @villares for translation

**Pull request merged**